modules/theme-tools/compat/twentytwenty.css

```diff
@@ -0,0 +1,130 @@
+/**
+ * Infinite scroll
+ */
+
+/* Globally hidden elements when Infinite Scroll is supported and in use. */
+.infinite-scroll .pagination-wrapper,
+.infinite-scroll.neverending .footer-nav-widgets-wrapper,
+.infinite-scroll.neverending #site-footer {
+	/* Theme Footer (when set to scrolling) */
+	display: none;
+}
+
+ /* When Infinite Scroll has reached its end we need to re-display elements that were hidden (via .neverending) before. */
```

Comment:
Is this normal?

Reply:
Yes, that's how we've handled it so far in most default themes.